server/models/index.js

```diff
@@ -0,0 +1,48 @@
+const fs = require("fs");
+const path = require("path");
+const Sequelize = require("sequelize");
+const envConfigs = require("../config/config");
+
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || "development";
+const config = envConfigs[env];
+const db = {};
+
+let sequelize;
+if (config.url) {
+  sequelize = new Sequelize(config.url, config);
+} else {
+  sequelize = new Sequelize(
+    config.database,
+    config.username,
+    config.password,
+    config
+  );
+}
+
+fs.readdirSync(__dirname)
```

Comment:
Do we really need to read model dynamically from directory here/?

Reply:
yes today we have small models but tomorrow if no of models increases this will come handy